engine/app/helpers/citizens_advice_cookie_preferences/application_helper.rb

```diff
@@ -2,5 +2,8 @@
 
 module CitizensAdviceCookiePreferences
   module ApplicationHelper
+    def cookies_preferences_page?
+      request.path == "/cookie-preferences/cookie_preference/edit"
+    end
```

Comment:
It would be good to find a way to avoid checking the path here as this depends on how the engine is mounted (side-note is the intention to flatten the paths at some point so it's just `/cookie-preferences/edit`?). Might be worth building on some of the work that mariana was doing with controller concerns, if we can shift this so the cookie preference controller defines:

```rb
def cookies_preference_page?
  true
end
```

Then a default method returning false in a controller helper exposed to the view that might avoid this. Works for now though 🙂

Reply:
Thanks, this is a great shout!  We've pushed up a refactor commit to swap out the approach